build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 ext {
     wordPressUtilsVersion = '2.7.0'
     wordPressLoginVersion = '0.18.0'
-    gutenbergMobileVersion = 'v1.81.1'
+    gutenbergMobileVersion = '5136-b4904076d045f99969d2a0dda6ae74630c8e1e95'
```

Comment:
Friendly reminder: we'll want to update this reference to the soon-to-be-created GB Mobile `1.82.2` tag.